### PR TITLE
Add more binning options for coordinates

### PR DESCRIPTION
**Pull request by ranquild**
Product doc https://www.notion.so/metabase/Allow-higher-granularities-for-lat-and-lon-binning-14d69354c90180ec8944f4543effd1e9

Adds `0.05`, `0.01`, `0.005` degrees binning options.

How to verify:
- New -> Question -> People
- Breakout -> Latitude -> Binning

Screenshot 2024-12-18 at 14 10 17
Screenshot 2024-12-18 at 14 10 27

Screenshot 2024-12-18 at 14 40 40

### CODE REVIEW

File: e2e/test/scenarios/binning/binning-options.cy.spec.js [hunk 1]
```diff
@@ -80,6 +80,9 @@ const LONGITUDE_BUCKETS = [
   "Bin every 1 degree",
   "Bin every 10 degrees",
   "Bin every 20 degrees",
+  "Bin every 0.05 degrees",
```
Comment (kamilmielnik) [inline e2e/test/scenarios/binning/binning-options.cy.spec.js:83]: Should we format the values according to "[Separator style](https://www.metabase.com/docs/latest/data-modeling/formatting#numbers)" setting?

File: e2e/test/scenarios/binning/correctness/shared/constants.js [hunk 2]
```diff
@@ -94,4 +94,16 @@ export const LONGITUDE_OPTIONS = {
     selected: "20°",
     representativeValues: ["180° W", "160° W", "100° W", "80° W", "60° W"],
   },
+  "Bin every 0.05 degrees": {
```
Comment (kamilmielnik) [inline e2e/test/scenarios/binning/correctness/shared/constants.js:97]: This arrow should be aligned to the right (it's a problem in `master`).

image

**Review by kamilmielnik — APPROVED** [review]
:+1:

**Review by lbrdnk — APPROVED** [review]

**Review by mngr — APPROVED** [review]

File: e2e/test/scenarios/binning/correctness/shared/constants.js (hunk 2 above)
Comment (uladzimirdev) [reply e2e/test/scenarios/binning/correctness/shared/constants.js:97]: yes, the problem comes from master and I'm working on a fix, but it's not easy

**Pull request merged**